truffle.js

```diff
@@ -15,6 +15,15 @@ module.exports = {
 		gasPrice: 1000,
 
     },
+    devUseHardcodedAddress: {
```

Comment:
better have a separate `serverdock/truffle.js` 
(the only entry you need in it is this added one, which is useless outside the docker..)